### DISCUSSION TITLE
update firmware on CI

### DISCUSSION THREAD

**Discussion by marcobrianza** [forkineye/ESPixelStick#661] [Q&A] [answered]
hello I am looking a way to update the firmware for an esp32 board flashed with a CI release.

The admin page of the web interface requires an .efu file.
where do I find it?

**Comment by MartinMueller2003** [comment, marked as answer]
You need to make one on your own. The flash tool has a button for creating an efu file. You do all of the same settings but instead of pressing upload you press the create efu button.

**Comment by marcobrianza** [comment]
great! i did not expect this!

**Comment by MartinMueller2003** [comment]
Did not expect what?

**Reply by marcobrianza** [reply]
to find a self service firmware packaging system.
